stratum/hal/lib/phal/onlp/onlp_wrapper.cc

```diff
@@ -156,6 +160,227 @@ ::util::StatusOr<SfpInfo> OnlpWrapper::GetSfpInfo(OnlpOid oid) const {
+      channel_mb = 0;
+      break;
+  case 2 || 34:
+      mb_mux.devaddr = 0x74;
+      channel_mb = 1;
+      break;
+  case 3 || 35:
+      mb_mux.devaddr = 0x74;
+      channel_mb = 2;
+      break;
+  case 4 || 36:
+      mb_mux.devaddr = 0x74;
+      channel_mb = 3;
+      break;
+  case 5 || 37:
+      mb_mux.devaddr = 0x74;
+      channel_mb = 4;
+      break;
+  case 6 || 38:
+      mb_mux.devaddr = 0x74;
+      channel_mb = 5;
+      break;
+  case 7 || 39:
+      mb_mux.devaddr = 0x74;
+      channel_mb = 6;
+      break;
+  case 8 || 40:
+      mb_mux.devaddr = 0x74;
+      channel_mb = 7;
+      break;
+  case 14 || 47:
+      mb_mux.devaddr = 0x75;
+      channel_mb = 0;
+      break;
+  case 13 || 48:
+      mb_mux.devaddr = 0x75;
+      channel_mb = 1;
+      break;
+  case 16 || 41:
+      mb_mux.devaddr = 0x75;
+      channel_mb = 2;
+      break;
+  case 15 || 42:
+      mb_mux.devaddr = 0x75;
+      channel_mb = 3;
+      break;
+  case 10 || 43:
+      mb_mux.devaddr = 0x75;
+      channel_mb = 4;
+      break;
+  case 9 || 44:
+      mb_mux.devaddr = 0x75;
+      channel_mb = 5;
+      break;
+  case 12 || 45:
+      mb_mux.devaddr = 0x75;
+      channel_mb = 6;
+      break;
+  case 11 || 46:
+      mb_mux.devaddr = 0x75;
+      channel_mb = 7;
+      break;
+  case 17 || 55:
+      mb_mux.devaddr = 0x76;
+      channel_mb = 0;
+      break;
+  case 18 || 56:
+      mb_mux.devaddr = 0x76;
+      channel_mb = 1;
+      break;
+  case 26 || 53:
+      mb_mux.devaddr = 0x76;
+      channel_mb = 2;
+      break;
+  case 25 || 54:
+      mb_mux.devaddr = 0x76;
+      channel_mb = 3;
+      break;
+  case 22 || 58:
+      mb_mux.devaddr = 0x76;
+      channel_mb = 4;
+      break;
+  case 21 || 57:
+      mb_mux.devaddr = 0x76;
+      channel_mb = 5;
+      break;
+  case 24 || 49:
+      mb_mux.devaddr = 0x76;
+      channel_mb = 6;
+      break;
+  case 23 || 50:
+      mb_mux.devaddr = 0x76;
+      channel_mb = 7;
+      break;
+  case 29 || 59:
+      mb_mux.devaddr = 0x77;
+      channel_mb = 0;
+      break;
+  case 30 || 60:
+      mb_mux.devaddr = 0x77;
+      channel_mb = 1;
+      break;
+  case 27 || 61:
+      mb_mux.devaddr = 0x77;
+      channel_mb = 2;
+      break;
+  case 28 || 62:
+      mb_mux.devaddr = 0x77;
+      channel_mb = 3;
+      break;
+  case 32 || 63:
+      mb_mux.devaddr = 0x77;
+      channel_mb = 4;
+      break;
+  case 31 || 64:
+      mb_mux.devaddr = 0x77;
+      channel_mb = 5;
+      break;
+  case 19 || 52:
+      mb_mux.devaddr = 0x77;
+      channel_mb = 6;
+      break;
+  default: //or case 20 || 51:
+      mb_mux.devaddr = 0x77;
+      channel_mb = 7;
+      break;
+  }
+  CHECK_RETURN_IF_FALSE(
+       ONLP_SUCCESS(onlp_functions_.onlp_i2c_mux_select(mb_mux, channel_mb)))
+       << "Failed to set MB MUX.";
+  // Change the page on slave 0x51 to access page 2
+  CHECK_RETURN_IF_FALSE(
+       ONLP_SUCCESS(onlp_functions_.onlp_i2c_write(0,0x51,0x7f,1,0x2,0)))
```

Comment:
0 is bus, mostly consistent
0x51 is the register (0x50 is for serial number/tmp/etc., 0x51 is register for accessing tunable parameters)
0x7f is the address, defined in the spec